sample/src/androidTest/java/com/github/grigory_rylov/instrumentaltestwithannotations/ExampleInstrumentedTest.java

```diff
@@ -18,7 +16,7 @@
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 @RunWith(AndroidJUnit4.class)
-@Feature(param = "+morda?param1=value1")
+@SampleAnnotation(stringParam = "+morda?param1=value1")
```

Comment:
If you removed this from this package, shouldn't it be imported from com.github.grishberg.annotationprinter?

Reply:
To use `@SampleAnnotation` from `com.github.grishberg.annotationprinter` I must move this annotation from `test` package into `src/main` package and make it public.